app/src/main/res/layout/fragment_pin_view.xml

```diff
@@ -156,5 +167,70 @@
             android:layout_marginStart="20dp"
             app:layout_constraintStart_toEndOf="@+id/pin_found_count"
             app:layout_constraintTop_toBottomOf="@+id/pin_caption" />
+
+        <ImageView
+            android:id="@+id/add_comment_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:adjustViewBounds="false"
+            android:background="@drawable/ic_circle"
+            android:backgroundTint="?attr/colorSecondary"
+            android:clickable="true"
+            android:focusable="true"
+            android:foreground="@drawable/ic_add_comment"
+            android:foregroundTint="?attr/colorOnSecondary"
+            android:scaleType="fitCenter"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
```

Comment:
Can we constrain the end of the caption text view to the start of this so we prevent it from overlapping? You may need to set the `layout_width` of the caption text view to `0dp` for it to work properly.